system-x/services/splunk/src/main/java/software/tnb/splunk/service/configuration/SplunkProtocol.java

```diff
@@ -0,0 +1,16 @@
+package software.tnb.splunk.service.configuration;
+
+public enum SplunkProtocol {
+    HTTP("http"),
+    HTTPS("https");
+
+    private final String value;
```

Comment:
seems like you don't need the value, you can create `toString` (or a custom method if you prefer) that does `name().toLowerCase()`